external/fv3fit/fv3fit/reservoir/readout.py

```diff
@@ -48,7 +49,7 @@ def batch_update(self, X, y):
     def get_weights(self):
         # use_least_squares_solve is useful for simple test cases
         # where np.linalg.solve encounters for singular XT.X
-
```

Comment:
`git checkout master external/fv3fit/fv3fit/reservoir/readout.py` to completely remove all changes to this file from pushing to master.